src/components/LegalBanner.tsx

```diff
@@ -72,17 +82,24 @@ const LegalText: React.FC = () => (
   </Text>
 )
 
-const LegalBanner: React.FC<LegalBannerProps> = ({ startOpen, useFull, title }) => {
-  const [open, setOpen] = useState(startOpen)
+const DISCLAIMER = 'GP_DISCLAIMER'
```

Comment:
I think `DISCLAIMER` is too general. 

nit: Maybe `IS_CLOSED_PARAM` or sth like that.

Reply:
it's the key name

Reply:
That's my point. `DISCLAIMER` var should make a reference of what it is. I think it should have then the prefix/suffix `_KEY`